src/DataFixtures/ContentFixtures.php

```diff
@@ -129,4 +141,36 @@ private function getValuesforFieldType($name, $field)
 
         return $data;
     }
+
+    private function getPresetRecords(): array
+    {
+        $records['blocks'][] = [
+            'title' => 'About',
+        ];
+        $records['blocks'][] = [
+            'title' => 'Search',
+        ];
+
+        $records['tests'][] = [
+            'selectfield' => 'bar',
+            'multiselect' => 'Michelangelo',
+            'slug' => 'title-of-the-test',
+            'title' => 'Title of the test',
+            'text_markup' => 'Text with <em>markup allowed</em>.',
+            'text_plain' => 'Text with <strong>no</strong> markup allowed.',
+        ];
+
+        return $records;
```

Comment:
return new Collection($records);

Reply:
Nah, in this instance (arrays in arrays), using Collection just makes it more cumbersome and less legible. Maybe (if/when) we refactor to using Bolt\Collection we might, but for now i'd like to leave it as is.